exercise3/block.py

```diff
@@ -31,14 +31,14 @@ def __init__(
     def hash(self) -> bytes:
         """
         TODO: Oblicz hash bloku wykorzystując do tego funkcję `hash` z modułu simple_cryptography.
-            Hash powinien składać się z wszystkich składowych bloku:
+            Należy obliczyć hasha skonkatenowanych składowych:
```

Comment:
Pomysł z dodaniem słowa `konkatenacja` jak najbardziej na +, ale trochę zmodyfikowałbym to zdanie
`Hash powinien składać się ze skonkatenowanych składowych bloku:`

Reply:
Imo to jest ciut ambigious wtedy, bo nie wiadomo, czy jest to konkatenacja hashów, czy hash z konkatenacji.
Skłaniałbym się bardziej w kierunku `Hash powinien zostać obliczony ze skonkatenowanych składowych bloku:`